js/apps/admin-ui/src/groups/Members.tsx

```diff
@@ -41,7 +42,7 @@ const MemberOfRenderer = (member: MembersOf) => {
     <>
       {member.membership.map((group, index) => (
         <>
-          <GroupPath key={group.id} group={group} />
+          <GroupPath key={group.id + "-" + member.id} group={group} />
```

Comment:
Is `member.id` not unique here? The `key` prop only has to be unique within the part of the hierarchy it's in, so perhaps we can get away with:
```suggestion
          <GroupPath key={member.id} group={group} />
```

If not, we can use a template string here:
```suggestion
          <GroupPath key={`${group.id}-${member.id}`} group={group} />
```